src/libs/EmojiUtils.js

```diff
@@ -170,8 +169,9 @@ function mergeEmojisWithFrequentlyUsedEmojis(emojis) {
 }
 
 /**
- * Update the frequently used emojis list by usage and sync with API
+ * Get the updated frequently used emojis list by usage
  * @param {Object|Object[]} newEmoji
+ * @return {Object[]}
  */
 function addToFrequentlyUsedEmojis(newEmoji) {
```

Comment:
I think the name `addToFrequentlyUsedEmojis` isn't valid anymore, we could rename it to `getFrequentlyUsedEmojis`.